src/hive_mind_bridge/launch/hive_mind_bridge.launch

```diff
@@ -1,8 +1,11 @@
 <?xml version="1.0" ?>
 <launch>
+    <arg name="robot_name" default="pioneer_0"/>
```

Comment:
The robot name should never have a 0 since it is reserved for the broadcast. replace with pioneer_1